src/main/java/org/openstreetmap/atlas/utilities/testing/TestAtlasHandler.java

```diff
@@ -91,25 +103,33 @@ public static Atlas getAtlasFromJsomOsmResource(final boolean josmFormat,
      *
      * @param pbfResource
      *            The pbf input resource to use
+     * @param iso
+     *            ISO code to be applied to all features
      * @return the resulting Atlas
      */
-    private static Atlas buildAtlasFromPbf(final Resource pbfResource)
+    private static Atlas buildAtlasFromPbf(final Resource pbfResource, final String iso)
     {
         // Create raw Atlas
         final AtlasLoadingOption loadingOption = AtlasLoadingOption.withNoFilter();
+        if (!iso.equals(ISOCountryTag.UNKNOWN_ISO_COUNTRY))
+        {
+            loadingOption.setCountrySlicing(true);
+            loadingOption.setCountryBoundaryMap(CountryBoundaryMap
+                    .fromBoundaryMap(Collections.singletonMap(iso, MultiPolygon.MAXIMUM)));
+        }
         final Atlas rawAtlas = new RawAtlasGenerator(pbfResource, loadingOption,
                 MultiPolygon.MAXIMUM).build();
 
-        // Way-section
-        return new WaySectionProcessor(rawAtlas, loadingOption).run();
+        // Country Slice and Way-Section
+        return new WaySectionProcessor(iso.equals(ISOCountryTag.UNKNOWN_ISO_COUNTRY) ? rawAtlas
+                : new RawAtlasCountrySlicer(loadingOption).slice(rawAtlas), loadingOption).run();
     }
 
-    private static Map<String, String> mergeTags(final String[] firstTags,
-            final String[] secondTags)
+    private static String[] mergeTags(final String[] firstTags, final String[] secondTags)
     {
         final List<String> allTags = new ArrayList<>(Arrays.asList(firstTags));
         allTags.addAll(Arrays.asList(secondTags));
-        return parseTags(allTags.toArray(new String[0]));
+        return allTags.toArray(new String[0]);
```

Comment:
Why the change here?

Reply:
Because of the addition of the iso argument to parseTags() I would have had to pass the iso through to this function. The only thing currently using this function is addArea(), so I thought it was cleaner to have parseTags() called on the result of this function from addArea().